### DISCUSSION TITLE
Knowledge cutoff dates

### DISCUSSION THREAD

**Discussion by varad0309** [QwenLM/Qwen3#1093] [Q&A]
Hi @jklj077 , can you please share the exact (month year) of knowledge cutoff of both Qwen2.5 and Qwen2.5-Coder.
This is what I got from the internet, but I am not sure if that's reliable.
Qwen2.5 : October 2023
Qwen2.5-Coder : June 2024

**Comment by jklj077** [comment]
ping @liudayiheng for Qwen2.5 and @huybery for Qwen2.5-Coder

**Comment by varad0309** [comment]
Thank you! @liudayiheng and @huybery , can you please confirm this? Thank you !

**Comment by sh0416** [comment]
Any updates?

**Comment by ArthurConmy** [comment]
Any update @huybery ? https://llm-stats.com/models/qwen-2.5-coder-32b-instruct claims March 2024 for another different date...

**Reply by sh0416** [reply]
I think we take a knowledge cutoff conservatively. In my opinion, June is appropriate for us to use.

**Comment by ghost** [comment]
Can anyone with reliable knowledge from the community about this please post an update or a link or something?

It would be really helpful to know this before using the model, I don't wanna use search feature for every single task because it may not works as well as one would reasonably expect.

And are there any models in the Qwen2.5 family that have a later cutoff date? Just curious. I want to consider exploring it.

---

Manish
Check out my kind of oldfangled, kinda newfangled, kinda cool, inevitably thought-provoking profile. And the limited mentorship opportunities for your no matter the non business major you have chosen be it - Bachelor's or Master's or Doctorates or Post-Doctorates.
In short, if you want to do check out my Readme.md at https://github.com/ket395 or https://github.com/ket395/ket395. It's quite good, I declare.

**Reply by ghost** [reply]
Please see https://github.com/QwenLM/Qwen2.5/issues/525#issuecomment-2159944330 for an estimate.

If you can't open the link - it's late 2023 for some subset of the training data.
